package/libnetwork/src/libnetwork.c

```diff
@@ -390,7 +412,10 @@ void ntrip_download(const network_config_t *config)
 
   struct curl_slist *chunk = ntrip_init(curl);
 
+  curl_easy_setopt(curl, CURLOPT_PUT,              1L);
+  curl_easy_setopt(curl, CURLOPT_CUSTOMREQUEST,    "GET");
   curl_easy_setopt(curl, CURLOPT_URL,              config->url);
+  curl_easy_setopt(curl, CURLOPT_READFUNCTION,     network_download_read);
```

Comment:
Should this be `network_upload_write`? Not sure if I'm understanding how the GGA put actually gets triggered

Reply:
In libcurl, the direction goes the other way. The read functions means "give me something to read to upload". The write functions mean "I'm going to write something I've downloaded". So here this means give NMEA sentences to read that will get uploaded. I'd keep the `network_download_` prefix associated with the download function here (even though, confusingly, the download is doing some upload as well).